files/settings-default.cfg

```diff
@@ -1,244 +1,709 @@
+# sneaking may find the character easier to control with this option
+# enabled.
+toggle sneak = false
+
+# This setting continues to be loaded and saved, but has no known
+# effect.  Presumably it and a related but also removed option named
+# "ui y sensitivity" used to control mouse sensitivity while in GUI
+# mode.  The default value is 1.0.
+ui sensitivity = 1.0
 
 [Map]
-# Adjusts the scale of the global map
+
+# It is not currently possible to control how many adjacent cells are
+# displayed in the map.  It appears that this is hardcoded to one
+# adjacent cell (3x3) in the code.  These settings control the canvas
+# and resolution sizes, and therefore the amount of panning required
+# to see the entire map, and the level of detail visible.
+
+# This integer setting adjusts the scale of the world map in the GUI
+# mode map display. The value is the width in pixels of each cell in
+# the map, so larger values result in larger more detailed world maps,
+# while smaller values result in smaller less detailed world maps.
+# However, the native resolution of the map source material appears to
+# be 9 pixels per unexplored cell and approximately 18 pixels per
+# explored cell, so values larger than 36 don't produce much
+# additional detail.  Similarly, the size of place markers is
+# currently fixed at 12 pixels, so values smaller than this result in
+# overlapping place markers.  Values from 12 to 36 are recommended.
+# For reference, Vvardenfell is approximately 41x36 cells.
 global map cell size = 18
 
+# This integer setting controls the zoom level for the HUD map display
+# (the map in the lower right corner while not in GUI mode).  A value
+# of 64 results in the HUD map displaying one exterior cell.  Since
+# the GUI mode map displays 3x3 cells, a value of approximately 21
+# displays the same area as the GUI mode map.  Larger values increase
+# the level of zoom, while smaller values are wasteful.
+#
+# Note that the actual size of the widget is always the same on the
+# screen unless the "scaling factor" setting in the "GUI" section is
+# changed.  Increasing both the scaling factor of the GUI and this
+# setting does result in a higher resolution HUD map, but
+# unfortunately with a scaled direction pointer on top of it.
+local map hud widget size = 256
+
+# This integer setting controls the resolution of the GUI mode local
+# map widget.  Larger values generally increase the visible detail in
+# map.  If this setting is half the "local map widget size" or
+# smaller, the map will generally be be fairly blurry.  Setting the
+# both options to the same value results in a map with good detail.
+# Values that exceed the "local map widget size" setting by more than
+# a factor of two are unlikely to provide much of an improvement in
+# detail since they're subsequently scaled back to the approximately
+# the map widget size before display.  The video resolution setting
+# interacts with this setting in that regard.
```

Comment:
Please note that larger map resolutions result in longer loading times, because the map is rendered on-demand when you enter a new cell. Also note that graphics cards have resolution limits, so the game may crash for abnormally large sizes or run out of VRAM.